CONTRIBUTING.md

```diff
@@ -29,7 +72,7 @@ popravi vse napake). Enako velja za spremembe med procesom pregleda.
 
 Po prvem uspešno sprejetem PRju boste tudi dodani na seznam razvijalcev.
 
-## Dodajanje novih problemov
+## SMERNICE ZA DODAJANJE NOVIH PROBLEMOV
```

Comment:
Tole govori o isti stvari kot odstavek zgoraj. Predlagam, da se zgornjega odnese dol, ker gre za specifično funkcionalnost in je treba najprej povedati, kako se naredi PR.